steps/version-extractor-android/0.0.3/step.yml

```diff
@@ -0,0 +1,68 @@
+title: Get versionName and versionCode on Android
+summary: |
+  Extract the versionName and the versionCode on an Android project using a Gradle task
+description: |
+  Extracts the version name and the version code from the Android project. It is useful if the versionName and versionCode values are defined somewhere else. Therefore, this step uses a Gradle task to print these two values. It supports only Groovy files for now.
+website: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android
+source_code_url: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android
+support_url: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android/issues
+published_at: 2020-05-04T16:52:13.68543+02:00
+source:
+  git: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android.git
+  commit: 80e6dc851f0395bc845ab0e3829dfa102ae2857c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_gradle_path: $BITRISE_SOURCE_DIR/app/build.gradle
+  opts:
+    is_required: true
+    summary: Path to the app's build.gradle file where the versionCode and versionName
+      are defined.
+    title: Path to the build.gradle file
+- gradlew_path: $GRADLEW_PATH
```

Comment:
Would recommend setting this to `./gradlew` or an empty string as $GRADLEW_PATH is not set by default. I found it used in the default gradle-runner step, but I believe it to be there by mistake.

Reply:
Sure, I'll change the default value to `./gradlew` 👍